pytradfri/api/aiocoap_api.py

```diff
@@ -15,55 +17,74 @@
     RequestTimedOut,
 )
 from aiocoap.numbers.codes import Code
+from aiocoap.protocol import BlockwiseRequest
+
+from pytradfri.command import Command
```

Comment:
Should we move this to a relative import?

Reply:
Yes, that should work and is more consistent.